src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -82,6 +82,7 @@
        (:loaded-method @state)
        [comps/EntityDetails {:entity (:loaded-method @state)
                              :editing? (:editing? props)
+                             :parse-error? (:parse-error? props)
```

Comment:
I'm all about names that are as descriptive as possible, so I'd go for `wdl-parse-error?` myself, but given that we know what we're talking about here, it's not really a huge deal.